BaseLib/ConfigTree.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef BASELIB_CONFIGTREE_H_
+#define BASELIB_CONFIGTREE_H_
+
+#include <string>
+#include <boost/property_tree/ptree.hpp>
```

Comment:
:+1:
Besides the question: Does it make sense to split off a forward declaration part from `BaseLib/ConfigTree.h`?